packages/tools/client-debugger/client-debugger/src/test/Utilities.ts

```diff
@@ -75,21 +76,29 @@ class MockContainer
 {
 	public readonly audience: IAudience = new MockAudience();
 
-	private _connectionState: ConnectionState = ConnectionState.Disconnected;
```

Comment:
I think we still want to leave this state tracking, and make sure the `connect` / `disconnect` still update it accordingly. Other tests will likely want to use this in the future.

Reply:
Make sense. Let me put them back.